### ISSUE TITLE
 git_mwindow_files_init racy

### ISSUE THREAD

**Issue by hanwen** [libgit2/libgit2#3875]
git_odb_backend_pack() calls git_mwindow_files_init without holding the lock.

**Comment by hanwen** [comment]
Ping? This is a serious issue: my application opens up to 900 repos in parallel, and I see crashes due to this.

**Comment by carlosmn** [comment]
Fixed via #3877

**Comment by hanwen** [comment]
thanks for the fix. It would be nice to have a stable release that includes this fix, either a 0.25 or 0.24.2 would be great.
